specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -263,6 +263,8 @@ function MergeButton<SCHEMA extends AnySchema>({
   const [formId] = React.useState(id('form'));
   const [saveBlocked, setSaveBlocked] = React.useState(false);
   const [showSaveBlockedDialog, setShowBlockedDialog] = React.useState(false);
+  const [warningDialog, _, handleCloseWarningDialog, toggleWarningDialog] =
```

Comment:
(optional) rename `toggleWarningDialog` to `handleToggleWarning` for consistency with how handlers are usually called?